clang/lib/CConv/Constraints.cpp

```diff
@@ -510,9 +502,12 @@ bool Constraints::graph_based_solve(ConstraintSet &Conflicts) {
 // the system is solved. If the system is solved, the first position is
 // an empty. If the system could not be solved, the constraints in conflict
 // are returned in the first position.
-std::pair<Constraints::ConstraintSet, bool> Constraints::solve() {
+// TODO: Returns copy of conflicts set. Can we either get rid of this return
+//       (It's not used as far as I can tell), or dynamically allocate and
+//       return a pointer.
+std::pair<std::set<VarAtom *>, bool> Constraints::solve() {
```

Comment:
If we are not using this, I would drop it. We can come back and change it later if we find a use for it.